client/index.html

```diff
@@ -0,0 +1,30 @@
+<html>
```

Comment:
https://blog.postman.com/postman-now-supports-socket-io/
maybe not worth doing html stuff?

Reply:
not insisting, just FYI